### PR TITLE
Feature/flho 341 club name

### PR DESCRIPTION
**Pull request by bendiggle**
* Added the step to the route config,
* Added the field to the fields config,
* Added the translations,
* Added the acceptance tests for the activity step and club-name step.
* Added page helpers for activity, club-name and club-postcode step.

### CODE REVIEW

File: apps/shooting-clubs/acceptance/features/activity.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const steps = require('../../');
+
+Feature('Activity step');
+
+Before((
+  I,
+  activityPage
+  ) => {
+    I.visitPage(activityPage, steps);
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/features/activity.js]: indentation

File: apps/shooting-clubs/acceptance/features/club-name.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Before((
+  I,
+  clubNamePage
+) => {
+  I.visitPage(clubNamePage, steps)
+});
+
+Scenario('The correct form element appears', (
+  I,
+    clubNamePage
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/features/club-name.js]: indentation

File: apps/shooting-clubs/acceptance/features/club-name.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  ) => {
+    I.seeElement(clubNamePage.clubName);
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/features/club-name.js]: indentation

File: apps/shooting-clubs/acceptance/features/club-name.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  }
+);
+
+Scenario('I see an error when I submit an empty form', (
+  I,
+    clubNamePage
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/features/club-name.js]: indentation

**Comment by bendiggle** [comment]
updated friend @joefitter

File: codecept.conf.js [hunk 5]
```diff
@@ -2,41 +2,46 @@
...
+    secondAuthorityHoldersAddressPage: dealerPagesPath('second-authority-holders-address.js'),
+    storageAddressPage: dealerPagesPath('storage-address.js'),
+    contactPage: dealerPagesPath('contact.js'),
+    contactDetailsPage: dealerPagesPath('contact-details.js'),
+    contactAddressPage: dealerPagesPath('contact-address.js'),
+    contactAddressAuthorityHolderPage: dealerPagesPath('contact-address-authority-holder.js'),
+    confirmPage: dealerPagesPath('confirm.js'),
+    confirmationPage: dealerPagesPath('confirmation.js'),
+    clubNamePage: shootingClubPagesPath('club-name.js'),
+    clubPostCodePage: shootingClubPagesPath('club-postcode.js')
```
Comment (joefitter) [inline codecept.conf.js]: this is becoming a pain to maintain. We should generate this list by recursing over the directory methinks

**Pull request merged**